mesh/nurbs.hpp

```diff
@@ -188,7 +188,10 @@ class NURBSExtension
    Mesh *patchTopo;
    int own_topo;
    Array<int> edge_to_knot;
+   // Set of knotvectors only containing unique knotvectors
    Array<KnotVector *> knotVectors;
+   // Extended set of knotvectors
```

Comment:
Can this comment be elaborated? What does "extended" mean in this context, and how are these knotvectors different from the original?

Reply:
Hi @dylan-copeland, I tried to explain this in the description of this PR. This problably wat not clear enough. I have updated the pr description to create clarification. I think the choice for the name "extended" is a bit off. I think that "comprehensive set" is a better name. I will update the name and add documentation.